<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>quickSwitchMode1</class>
 <widget class="QDialog" name="quickSwitchMode1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>391</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="6">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="7">
    <widget class="QLabel" name="label_Att4">
     <property name="text">
      <string>Аттенюатор, dB</string>
     </property>
    </widget>
   </item>
   <item row="7" column="9" colspan="2">
    <widget class="QPushButton" name="sendButton">
     <property name="text">
      <string>Старт</string>
     </property>
    </widget>
   </item>
   <item row="1" column="10">
    <widget class="QLabel" name="label_Oth6">
     <property name="text">
      <string>Прочее</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_Att1">
     <property name="text">
      <string>Аттенюатор, dB</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="connectButton">
     <property name="text">
      <string>Подключить</string>
     </property>
    </widget>
   </item>
   <item row="1" column="8" colspan="2">
    <widget class="QLabel" name="label_Fv5">
     <property name="text">
      <string>Фазовращатель, °</string>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <layout class="QVBoxLayout" name="verticalLayoutAtt_2"/>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayoutAtt_1"/>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_Fv2">
     <property name="text">
      <string>Фазовращатель, °</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="disconnectButton">
     <property name="text">
      <string>Отключить</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_24">
     <item row="0" column="0">
      <widget class="QComboBox" name="comboBoxToolList">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtBottom</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButtonRefresh">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Обновить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QVBoxLayout" name="verticalLayoutFv_1"/>
   </item>
   <item row="2" column="8" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayoutFv_2"/>
   </item>
   <item row="2" column="10">
    <layout class="QVBoxLayout" name="verticalLayoutOther_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
    </layout>
   </item>
   <item row="7" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Интервал, мс</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_22">
     <item row="0" column="1">
      <widget class="QLCDNumber" name="lcdNumber_12">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="digitCount">
        <number>7</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLCDNumber" name="lcdNumber_11">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="digitCount">
        <number>7</number>
       </property>
       <property name="value" stdset="0">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="7" colspan="3">
    <layout class="QGridLayout" name="gridLayout_23">
     <item row="0" column="1">
      <widget class="QLCDNumber" name="lcdNumber_22">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="digitCount">
        <number>7</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLCDNumber" name="lcdNumber_21">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="digitCount">
        <number>7</number>
       </property>
       <property name="value" stdset="0">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="2">
    <layout class="QVBoxLayout" name="verticalLayoutOther_1"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_Oth3">
     <property name="text">
      <string>Прочее</string>
     </property>
    </widget>
   </item>
   <item row="7" column="8">
    <widget class="QPushButton" name="stopButton">
     <property name="text">
      <string>Стоп</string>
     </property>
    </widget>
   </item>
   <item row="7" column="5">
    <widget class="QSpinBox" name="spinBox">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="4" column="7">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QTextEdit" name="textEdit">
     <property name="minimumSize">
      <size>
       <width>670</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="0" column="10">
    <widget class="QPushButton" name="PortButton">
     <property name="text">
      <string>Порт</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
